Nagios 4 query handler : nerd

Hello, 
when I submitted the following commands to test query handler
echo -e '@nerd list\0'|nc -U /usr/local/nagios/var/rw/nagios.qh
I got a response :
404: nerd: No such handler
Install nagios 4 according to your [quickstart documents:
https://support.nagios.com/kb/article/nagios-core-installing-nagios-core-from-source-96.html#Ubuntu
Any suggestions ???
Regards,
dbaid

Hello,

   ./configure --enable-nerd 
    and make all
   I got errors

gcc -Wall -g -O2 -DHAVE_CONFIG_H -c skiplist.c -o skiplist.o
gcc -Wall -g -O2 -DHAVE_CONFIG_H -c nsock.c -o nsock.o
gcc -Wall -g -O2 -DHAVE_CONFIG_H -c nspath.c -o nspath.o
ar cr libnagios.a squeue.o kvvec.o iocache.o iobroker.o bitmap.o dkhash.o runcmd.o nsutils.o fanout.o pqueue.o worker.o skiplist.o nsock.o nspath.o
make[2]: Leaving directory '/tmp/nagioscore-nagios-4.4.5/lib'
gcc -Wall -I.. -g -O2 -DHAVE_CONFIG_H -DNSCORE -o nagios nagios.o broker.o nebmods.o ../common/shared.o nerd.o query-handler.o workers.o checks.o config.o commands.o events.o flapping.o logging.o macros-base.o netutils.o notifications.o sehandlers.o utils.o retention-base.o xretention-base.o comments-base.o xcomments-base.o objects-base.o xobjects-base.o statusdata-base.o xstatusdata-base.o perfdata-base.o xperfdata-base.o downtime-base.o  -Wl,-export-dynamic   -lm  -ldl  ../lib/libnagios.a
nagios.o: In function `main':
/tmp/nagioscore-nagios-4.4.5/base/nagios.c:688: undefined reference to `nerd_init'
collect2: error: ld returned 1 exit status
Makefile:157: recipe for target 'nagios' failed
make[1]: *** [nagios] Error 1
make[1]: Leaving directory '/tmp/nagioscore-nagios-4.4.5/base'
Makefile:88: recipe for target 'all' failed

--- a/base/nerd.c
+++ b/base/nerd.c
@@ -1,4 +1,3 @@
-#ifdef ENABLE_NERD
 /*
  * Nagios Event Radio Dispatcher
  *
@@ -26,6 +25,8 @@
 #include "include/nebmods.h"
 #include "include/nebmodules.h"
 #include "include/nebstructs.h"
+
+#ifdef ENABLE_NERD
 
 struct nerd_channel {
 	const char *name; /* name of this channel */

--- a/configure.ac
+++ b/configure.ac
@@ -459,7 +459,7 @@ AC_ARG_ENABLE(nerd,
 	NERD=yes
 )
 if test "x$NERD" = "xyes"; then
-	AC_DEFINE_UNQUOTED(ENABLE_NERD,,[defined if user enabled NERD (radio dispatcher)])
+	AC_DEFINE(ENABLE_NERD,1,[defined if user enabled NERD (radio dispatcher)])
 	NERD_O="nerd.o"
 fi
 AC_SUBST(NERD_O)

